deploy/kubernetes/tracee.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app.kubernetes.io/name: tracee
+    app.kubernetes.io/instance: tracee-v1
+    app.kubernetes.io/version: "v0.5.0"
```

Comment:
I'm not sure how to manage the version with static yamls. also the image ref is latest, so let's remove it